### PR TITLE
New: Begin the section on the logs directory

### PR DESCRIPTION
**Pull request by ahankinson**
This PR adds some text about the 'logs' directory.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,6 +180,17 @@ <h2>Object Conformance Declaration</h2>
...
     <section id="logs-directory">
         <h2>Logs Directory</h2>
+        <p>
+            The base directory of the object MUST contain a logs directory, which MAY be empty. Implementers SHOULD use
+            this for storing files that log any actions taken on the object. Since these logs may be subject to local
+            standards adherence needs, the format of these logs is considered out-of-scope for the OCFL Object. However,
+            clients operating on the object SHOULD log actions here that are not otherwise captured.
+        </p>
+        <p>
+            The purpose of the logs directory is to provide implementers with a location for storing local information
```
Comment (awoods) [inline index.html]: Should this be a non-normative block?

File: index.html [hunk 2]
```diff
@@ -180,6 +180,17 @@ <h2>Object Conformance Declaration</h2>
...
+            about the object that may not necessarily be part of the object itself, and to expand on the processable
+            information that can be found in the object's inventory.
```
Comment (awoods) [inline index.html]: > expand on the processable information that can be found in the object's inventory.

I am a little unclear on the meaning of the above.

Comment (neilsjefferies) [reply index.html]: Could we include an example: such as a fixity check that finds nothing wrong?

File: index.html (hunk 1 above)
Comment (neilsjefferies) [reply index.html]: The fact that there MUST be a logs directory is normative - even if the contents are unspecified.

Comment (awoods) [reply index.html]: Agreed. The first block is normative ( https://github.com/OCFL/spec/pull/45/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR183 ), and 
the second block seems non-normative ( https://github.com/OCFL/spec/pull/45/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR190 ).

I would suggest wrapping the second block as a non-normative statement:
```
      <blockquote class="informative">
        <p>
          Non-normative note:
          The purpose of the logs directory is to provide implementers with a location for storing local information
          about the object that may not necessarily be part of the object itself, and to expand on the processable
          information that can be found in the object's inventory.
        </p>
      </blockquote>
```

File: index.html (hunk 2 above)
Comment (rosy1280) [reply index.html]: Answer from the editors meeting: you could do it that way, which is why it can be empty.

**Comment by ahankinson** [comment]
A directory named `logs` is permissible, but it's not a requirement.

**Comment by ahankinson** [comment]
Changes added. Will need re-approval.

**Comment by zimeon** [comment]
@ahankinson - need to fix line length to keep travis happy

**Comment by ahankinson** [comment]
@awoods could you merge this please?

**Comment by zimeon** [comment]
@ahankinson I think per our rules the _next_ :+1: can merge

**Comment by ahankinson** [comment]
OK; It would just be nice to get this one done and merged.

**Comment by awoods** [comment]
@neilsjefferies ? @rosy1280 ? @julianmorley ? are any of you available for a review/merge?

**Pull request merged**